cmd/elkey/README.md

```diff
@@ -0,0 +1,19 @@
+## elkey
```

Comment:
is el here for eigenlayer or execution layer?
Can't use el for eigenlayer like this, since its extremely confusing.
We've been told this by EF as well afaik.
maybe `egnkey`?

Reply:
`egnkey` it is